dora/core/client/fs/src/main/java/alluxio/client/file/DoraFileOutStreamBak.java

```diff
@@ -0,0 +1,347 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.client.file;
+
+import alluxio.AlluxioURI;
+import alluxio.Constants;
+import alluxio.client.AlluxioStorageType;
+import alluxio.client.UnderStorageType;
+import alluxio.client.block.BlockStoreClient;
+import alluxio.client.block.policy.options.GetWorkerOptions;
+import alluxio.client.block.stream.BlockOutStream;
+import alluxio.client.block.stream.UnderFileSystemFileOutStream;
+import alluxio.client.file.options.OutStreamOptions;
+import alluxio.conf.AlluxioConfiguration;
+import alluxio.conf.PropertyKey;
+import alluxio.exception.ExceptionMessage;
+import alluxio.exception.PreconditionMessage;
+import alluxio.exception.status.UnavailableException;
+import alluxio.grpc.CompleteFilePOptions;
+import alluxio.grpc.FileSystemMasterCommonPOptions;
+import alluxio.metrics.MetricKey;
+import alluxio.metrics.MetricsSystem;
+import alluxio.resource.CloseableResource;
+import alluxio.retry.ExponentialTimeBoundedRetry;
+import alluxio.retry.RetryPolicy;
+import alluxio.util.CommonUtils;
+import alluxio.util.FileSystemOptionsUtils;
+import alluxio.wire.BlockInfo;
+import alluxio.wire.OperationId;
+import alluxio.wire.WorkerNetAddress;
+
+import com.codahale.metrics.Counter;
+import com.codahale.metrics.Timer;
+import com.google.common.base.Preconditions;
+import com.google.common.io.Closer;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import java.util.UUID;
+import javax.annotation.concurrent.NotThreadSafe;
+import javax.annotation.concurrent.ThreadSafe;
+
+/**
+ * Provides a streaming API to write a file. This class wraps the BlockOutStreams for each of the
+ * blocks in the file and abstracts the switching between streams. The backing streams can write to
+ * Alluxio space in the local machine or remote machines. If the {@link UnderStorageType} is
+ * {@link UnderStorageType#SYNC_PERSIST}, another stream will write the data to the under storage
+ * system.
+ */
+@NotThreadSafe
+public class DoraFileOutStreamBak extends FileOutStream {
```

Comment:
`Bak` meaning `backup` or `backing` or sth else?

Reply:
It was old code which should be removed, I think.

Reply:
I forgot to delete this file.